en/docs/get-started/tutorials/create-your-first-rest-api-proxy.md

```diff
@@ -1,94 +1,121 @@
 # Create Your First REST API Proxy
 
-Choreo is a low-code cloud-native platform for building integration solutions. In this tutorial, you will do the following to explore REST API Proxies in Choreo:
+This tutorial introduces REST API proxies. 
+
+An API proxy fronts an API backend and intercepts the requests to the REST API. You can apply security, rate limiting, etc., to the REST API Proxy.
```

Comment:
This sentence is a little confusing.. it says it fronts the Bakcend,, and then it says it intercepts the requests to the REST API.  Shall we say, "An API Proxy exposes an existing API on Choreo and intercepts the requests to the REST API."

Reply:
Shall we say, " The API Proxy acts as a managed API to which you can apply security, rate-limiting, etc."

Reply:
Done. Did a minor change to the 2nd suggestion.